test/lint/blocking_commands.rb

```diff
@@ -1,5 +1,8 @@
 module Lint
   module BlockingCommands
+    TIMEOUT_SEC = 0.01
+    DELAY_SEC = 0.05
```

Comment:
Similar issue with the naming here. I'd almost suggest not to constantize this and to use `LOW_TIMEOUT * 5` as an expression.